src/components/__test__/LoggedHeader.test.js

```diff
@@ -0,0 +1,37 @@
+import LoggedHeader from '../LoggedHeader.vue';
+import { shallowMount, createLocalVue } from '@vue/test-utils';
+import Buefy from 'buefy';
+import mock from '../../test/mockBack';
+
+describe('LoggedHeader', () => {
+  let localVue;
+  const mock = { newsLetter: "Hi, I'm a new bar" };
+  beforeEach(() => {
```

Comment:
what sense does this function i dont get it. Explain pls

Reply:
It's for testing workflow, here the docs https://jestjs.io/docs/en/setup-teardown.html